crates/net/network/src/config.rs

```diff
@@ -358,6 +367,13 @@ impl NetworkConfigBuilder {
         self
     }
 
+    #[cfg(feature = "optimism")]
+    /// Sets whether tx gossip is disabled.
```

Comment:
nit - believe we are supposed to keep comments above attributes
```suggestion
    /// Sets whether tx gossip is disabled.
    #[cfg(feature = "optimism")]
```